spec/ruby/core/module/refine_spec.rb

```diff
@@ -241,9 +243,43 @@ def foo; "foo from singleton class"; end
       result.should == "foo from singleton class"
     end
 
+    it "looks in later prepended modules to the refined module first" do
+      a = Module.new do
+        def foo
+         "foo from A"
+        end
+      end
+
+      include_me_later = Module.new do
+        def foo
+          "foo from IncludeMeLater"
+        end
+      end
+
+      c = Class.new do
+        include a
+      end
+
+      refinement =
+        Module.new do
+          refine c do; end
+        end
+
+      result = nil
+      Module.new do
+        using refinement
+        c.include include_me_later
+        result = c.new.foo
+      end
+
+      result.should == "foo from IncludeMeLater"
```

Comment:
Could you move the `should ==` above inside the `Module.new`, so we don't need a `result` variable?

Reply:
We have the `result` var "style" for all other tests. I'm also not a fan of this approach, but for the consistency I suggest leaving it as it is.

Reply:
OK, that's fine then, I didn't notice in the diff.